index.html

```diff
@@ -11,54 +11,153 @@
+                            <div>2015</div>
+                        </div>
+                    </div>
+                    <div class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                    <div class="categories">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                
+                <div class="card">
+                    <div class="snapshot4"></div>
+                    <div class="project-title">Multi-Post
+                        Stories</div>
+                    <div class="project-subtitle">
+                        <div class="flex-container-tonic">
+                            <div class="canopy">CANOPY</div>
+                            <div class="dot"></div>
+                            <div>Back End Dev</div>
+                            <div class="dot"></div>
+                            <div>2015</div>
+                        </div>
+                    </div>
+                    <div class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                    <div class="categories">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
                 
-                <ul>
-                    <li><a href="https://twitter.com"><i class="fab fa-twitter"></i></a></li>
-                    <li><a href="https://facebook.com"><i class="fab fa-facebook"></i></a></li>
-                    <li><a href="https://linkedin.com"><i class="fab fa-linkedin-in"></i></a></li>
-                    <li><a href="https://github.com"><i class="fab fa-github"></i></a></li>
-                    <li><a href="https://youtube.com"><i class="fab fa-youtube"></i></a></li>
-                </ul>
+            
             </section>
-        </main>
-    </div>
-</body>
 
+            <section class="about">
+                <div class="headline">
+                    About<br>Myself
+                </div>
 
+                <div class="body">
+                    <p>
+                        I’m a software developer! I can help you build a product feature or website. Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                    </p>
+                </div>
 
+                <div class="social-media">
+                    <p>LET'S CONNECT</p>
```

Comment:
- [ ] the same thing here, It would be better to use a header tag hx (`<h2>`,`<h3>`, ...) instead of `<p>` 👌